go/oasis-test-runner/cmd/common/common.go

```diff
@@ -0,0 +1,26 @@
+// Package common contains common constants and variables.
+package common
+
+import (
+	"github.com/oasislabs/oasis-core/go/oasis-test-runner/scenario"
+)
+
+const (
+	CfgLogFmt   = "log.format"
+	CfgLogLevel = "log.level"
+	CfgTest     = "test"
+	CfgTestP    = "t"
+
+	MetricUp = "oasis_up"
+
+	// ScenarioParamsMask is the form of parameters passed to specific scenario.
+	//
+	// [1] parameter is test name, [2] parameter is parameter name. This is
+	// used when binding and parsing (recursive) parameters with viper.
+	ScenarioParamsMask = "%[1]s.%[2]s"
+)
+
+var (
+	Scenarios        = make(map[string]scenario.Scenario)
```

Comment:
It seems dangerous exposing these two directly. Instead, make them private and expose `RegisterScenario(s scenario.Scenario, default bool)`/`Scenarios`/`DefaultScenarios` functions.

Also mention in the documentation comments that these are not safe for concurrent use.